dialog_lib/agents/abstract.py

```diff
@@ -103,6 +112,110 @@ def messages(self):
         return self.memory.messages
 
 
+class AbstractLCEL(AbstractLLM):
```

Comment:
just a note for new ideas: I know `AbstractLLM` is a convention from the very beginning of dialog, but since this now lives under `agents`, maybe it'd be good to start naming classes here as _XPTOAgent_, which would still be chain wrappers, but making ground to implement real Agents in the future (with LangGraph). Of course, always keeping compatibility.

Reply:
@lgabs good question. Can we solve the naming after this PR? So we can make the change with more people's opinions